scripts/dumps/DumpAlignSlice.pl

```diff
@@ -103,15 +94,19 @@ =head2 SPECIFYING THE QUERY SLICE
 
 =item B<--seq_region region_name>
 
-Query region name, i.e. the chromosome name
+Query region name, i.e. the chromosome name. Default 13.
```

Comment:
```suggestion
Query region name, i.e. the chromosome name.
Default 13
```
Just for readability. Full stop removed for consistency.